src/main/kotlin/org/rust/ide/refactoring/move/common/RsMoveCommonProcessor.kt

```diff
@@ -98,12 +104,20 @@ class RsMoveCommonProcessor(
     private val codeFragmentFactory: RsCodeFragmentFactory = RsCodeFragmentFactory(project)
 
     private val sourceMod: RsMod = elementsToMove
+        .also { if (it.isEmpty()) throw IncorrectOperationException("No items to move") }
         .map { it.element.containingModStrict }
         .distinct()
         .singleOrNull()
         ?: error("Elements to move must belong to single parent mod")
 
     private val pathHelper: RsMovePathHelper = RsMovePathHelper(project, targetMod)
+    private lateinit var conflictsDetector: RsMoveConflictsDetector
```

Comment:
Will this field be used later outside `preprocessUsages`, or why it is not a local variable?

Reply:
Yes, to [make moved items public if necessary](https://github.com/intellij-rust/intellij-rust/blob/92b32a2758fab80c36abe8d69d97bb478c8b4a0e/src/main/kotlin/org/rust/ide/refactoring/move/common/RsMoveCommonProcessor.kt#L410)